### PR TITLE
Fix error on aarch64 with binutils2.35.1 and later versions.

### PR DESCRIPTION
**Pull request by leimaohui**
Signed-off-by: Lei Maohui <leimaohui@cn.fujitsu.com>

### CODE REVIEW

**Comment by leimaohui** [comment]
I'm sorry, but How to fix "CodeFactor — 6 issues found." ?

**Comment by honggyukim** [comment]
> I'm sorry, but How to fix "CodeFactor — 6 issues found." ?

No worries. You can just ignore them :)

**Comment by honggyukim** [comment]
Thanks. It looks good now, but could you change "Fixed: #1256" to "Fixed: #1254" in the commit message?

**Comment by leimaohui** [comment]
OK, I have added commit message. Please confirm.

> Thanks. It looks good now, but could you change "Fixed: #1256" to "Fixed: #1254" in the commit message?

Have fixed.

**Review by honggyukim — APPROVED** [review]
@leimaohui Thanks very much for this fix!

File: utils/symbol.c [hunk 1]
```diff
@@ -560,6 +560,7 @@ int load_elf_dynsymtab(struct symtab *dsymtab, struct uftrace_elf_data *elf,
 	}
 	else if (elf->ehdr.e_machine == EM_AARCH64) {
 		plt_addr += 16;    /* AARCH64 PLT0 size is 32 */
+		plt_entsize = 16;
```
Comment (namhyung) [inline utils/symbol.c:563]: can you add 'if (plt_entsize == 0)'?

**Comment by namhyung** [comment]
I don't like to have a separate commit for it.  Can you please update the original commit and force push it to the same branch?

**Review by leimaohui — COMMENTED** [review]
> I don't like to have a separate commit for it. Can you please update the original commit and force push it to the same branch?

Sorry, not very familiar with modifying PR. I have force updated these two commits into one. Please confirm again.

**Review by namhyung — APPROVED** [review]
LGTM, thanks for all your work!

**Pull request merged**